lang_js/analyze/ast_js.ml

```diff
@@ -272,8 +272,11 @@ and obj_ = property list bracket
 and class_ = { 
   (* usually simply an Id *)
   c_extends: expr option;
-  c_body: property list bracket;
+  c_body: c_body_biding list bracket;
 }
+  and c_body_biding =
+    | CBodyClassic of property
+    | CEllipsis of tok
```

Comment:
you can also add the (* sgrep-ext: *) comment before CEllipsis so reader can see that it's a construct not part
of the JS language but a sgrep specific construct.

Reply:
ok

Reply:
addressed